Sources/ClientRuntime/Networking/Checksums/ChecksumAlgorithm.swift

```diff
@@ -3,15 +3,18 @@
  * SPDX-License-Identifier: Apache-2.0.
  */
 
+import protocol SmithyChecksumsAPI.Checksum
+import enum SmithyChecksumsAPI.ChecksumAlgorithm
+import enum SmithyChecksumsAPI.HashResult
+import struct Foundation.Data
 import AwsCommonRuntimeKit
 
 public enum HashError: Error {
     case invalidInput
     case hashingFailed(reason: String)
 }
 
-public enum ChecksumAlgorithm {
-    case crc32, crc32c, sha1, sha256, md5
+extension ChecksumAlgorithm {
```

Comment:
ChecksumAlgorithm itself was moved out to `SmithyHTTPAuthAPI`.  This extension remains here because it is dependent on CRT.

Reply:
+1 to Chan's comment on moving them to their own module, but agree with keep ChecksumsCRT module in ClientRuntime as checksums themselves aren't AWS specific concepts